adapters/Fyber/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Fyber iOS Mediation Adapter Changelog
 
+#### Version 7.5.6.1
+- Verified compatibility with Fyber Marketplace SDK version 7.5.6.
+- Removed Fyber logs
```

Comment:
This might be misleading to think that all logging from the adapter has been removed.
Maybe revise to: "Temporarily removed Fyber Marketplace SDK-specific logging."